### PR TITLE
build:  Changes that affect the build system

### PR DESCRIPTION
**Pull request by Mikemosca**
build:  Changes that affect the build system

### CODE REVIEW

File: scripts/ruby_tasks.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+#
+# (C) Copyright IBM Corp. 2018, 2019.  All Rights Reserved.
```
Comment (mamoonraja) [inline scripts/ruby_tasks.sh:3]: this should be 2020

File: scripts/ruby-tasks/visual_recognition_v3.rake [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
```
Comment (mamoonraja) [inline scripts/ruby-tasks/visual_recognition_v3.rake:1]: we stopped using tasks for `vis rec`, so we can delete this one, and the patch file.

File: scripts/ruby_tasks.sh (hunk 1 above)
Comment (Mikemosca) [reply scripts/ruby_tasks.sh]: done

File: scripts/ruby-tasks/visual_recognition_v3.rake (hunk 2 above)
Comment (Mikemosca) [reply scripts/ruby-tasks/visual_recognition_v3.rake]: done

**Pull request merged**